tests/integration/test_tf_symbolic.py

```diff
@@ -23,10 +23,18 @@
 
 import numpy as np
 from scipy.special import factorial
-import tensorflow as tf
+
+try:
+    import tensorflow as tf
+except (ImportError, ModuleNotFoundError):
+    pytestmark = pytest.mark.skip("TensorFlow not installed")
+else:
+    if tf.__version__[:3] != "1.3":
+        pytestmark = pytest.mark.skip("Test only runs with TensorFlow 1.3")
```

Comment:
Does this cause all tests to then be skipped?

Reply:
Yep, since this is a module level attribute, this will skip the entire `test_tf_symbolic` module